### PR TITLE
Add spec via Google Calendar

### PR DESCRIPTION
**Pull request by k1LoW**

### CODE REVIEW

File: holidays.yml [hunk 1]
```diff
@@ -733,11 +732,11 @@
 2017-10-09: 体育の日
 2017-11-03: 文化の日
 2017-11-23: 勤労感謝の日
+2017-12-23: 天皇誕生日
```
Comment (k1LoW) [inline holidays.yml]: http://www8.cao.go.jp/chosei/shukujitsu/gaiyou.html

Comment (komagata) [inline holidays.yml]: 👍

**Comment by k1LoW** [comment]
@komagata Add spec && Fix '天皇誕生日' date.

**Comment by komagata** [comment]
@k1LoW Thank you additing spec.

**Pull request merged**